# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/src/DriverCrashGuard.cpp
# Commit: d158b20d9a65
# Full Hash: d158b20d9a65cd05dd69a586473216b800523083
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - PD: Allow WMF decoding inside RDD process. r=mattwoodrow
#   
#   This assumes that access to Windows WMF and DirectX is allowed from RDD process.
#   
#   Depends on D91699
# ==============================================================================

diff -r d0c9c64867cc -r d158b20d9a65 gfx/src/DriverCrashGuard.cpp
--- a/gfx/src/DriverCrashGuard.cpp	Tue Oct 20 23:26:49 2020 +0000
+++ b/gfx/src/DriverCrashGuard.cpp	Tue Oct 20 23:26:52 2020 +0000
@@ -59,9 +59,9 @@
 }
 
 static inline bool AreCrashGuardsEnabled(CrashGuardType aType) {
-  // Crash guard isn't supported in the GPU process since the entire
+  // Crash guard isn't supported in the GPU or RDD process since the entire
   // process is basically a crash guard.
-  if (XRE_IsGPUProcess()) {
+  if (XRE_IsGPUProcess() || XRE_IsRDDProcess()) {
     return false;
   }
 #ifdef NIGHTLY_BUILD